bionic/deriver.py

```diff
@@ -337,6 +340,9 @@ def _compute_result_group_for_dnode(self, dnode):
             self._get_or_create_task_state_for_key(task.keys[0]) for task in dinfo.tasks
         ]
 
+        if self._bootstrap is not None:
+            self._bootstrap.resize_executor()
```

Comment:
WDYT about doing this inside `Executor.submit`? The need for resizing seems to me to be a detail of the way we use Loky, and this would also simplify the deriver and remove some `is not None` checks.

Reply:
That seems a lot cleaner. Thanks!